### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by TheEmperor342**
### Changelog

- \+ Shorter Help Command
- \+ Welcomer and Leveling now have better cards
- \+Change Presence
- \- Deleted Music.py
- \- Changed everything (that could be changed) from json to SQLite

#### RIP Discord.py

### CODE REVIEW

**Comment by itsmebirdie** [comment]
Acknowledged.

**Comment by itsmebirdie** [comment]
can you submit a brief description of what you have done in this PR? This would help in for a faster code review :)

**Review by itsmebirdie — CHANGES_REQUESTED** [review]
## Testing failed
### Reason: Critical Error
*Image(s) attached for reference*:
image

*Error as plaintext*:
```
tester /workspace/discord-assistant $ python3 main.py
Enter The Token of your bot: REDACTED
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 606, in _load_from_module_spec
    spec.loader.exec_module(lib)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/workspace/discord-assistant/Cogs/Fun.py", line 4, in <module>
    from discord_components import Button, ButtonStyle, InteractionType
ImportError: cannot import name 'InteractionType' from 'discord_components' (/workspace/.pip-modules/lib/python3.8/site-packages/discord_components/__init__.py)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "main.py", line 58, in <module>
    bot.load_extension(f"Cogs.{i[:-3]}")
  File "/workspace/.pip-modules/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 678, in load_extension
    self._load_from_module_spec(spec, name)
  File "/workspace/.pip-modules/lib/python3.8/site-packages/discord/ext/commands/bot.py", line 609, in _load_from_module_spec
    raise errors.ExtensionFailed(key, e) from e
discord.ext.commands.errors.ExtensionFailed: Extension 'Cogs.Fun' raised an error: ImportError: cannot import name 'InteractionType' from 'discord_components' (/workspace/.pip-modules/lib/python3.8/site-packages/discord_components/__init__.py)
tester /workspace/discord-assistant $ 
```

**Comment by TheEmperor342** [comment]
It works on my machine though-

**Comment by itsmebirdie** [comment]
> It works on my machine though-

We aren't going to ship your machine in production :')

**Comment by TheEmperor342** [comment]
`$ pip uninstall discord-components`
`$ pip install discord-components==1.1.4`
`$ pip install easy-pil==0.0.2`

**Comment by TheEmperor342** [comment]
> can you submit a brief description of what you have done in this PR? This would help in for a faster code review :)

Basically all files were changed. Here are the main changes

- Leveling was completely rewritten
- Help command's size was decreased and moved to its own file
- Welcomer was completely rewritten
- all the other things writing data in JSON were changed to SQLite

**Comment by itsmebirdie** [comment]
Review request acknowledged.

**Comment by itsmebirdie** [comment]
Acknowledged.

**Comment by TheEmperor342** [comment]
> Acknowledged.

I have a bug to fix, (and a conflict) I don't know how that pushed a commit

**Review by itsmebirdie — APPROVED** [review]
Code testing was successful.
Approved for merge,
Good Job

**TLDR;** This is going to be the last Pull Request/Commit for Assistant in Python as we are moving to a better version at [Assistant in NodeJS](https://github.com/hackarmour/discord-assistant-js/) and will be archiving this repo in further announcements.
